modules/packages/UnitTest.chpl

```diff
@@ -7,15 +26,15 @@ Any function of the form
   proc funcName(test: borrowed Test) throws {}
 
 is treated as a test function. These functions must accept an object of Test
-Class. We use :proc:`~UnitTest.runTest` to pass the tests.
+Class. We use :proc:`~UnitTest.main()` to run the tests.
 
 .. note::
 
   Tests must be launched with a test launcher. Currently, this is a standalone application 
   built separately. However, this functionality will eventually be integrated into mason.
 
 Assert Functions
-================
+----------------
```

Comment:
I noticed that this list includes:

- skip
- skipIf

I wouldn't consider these assert functions. I think we want a separate list of functions that includes skips, numlocales checks, and dependencies, maybe call it Test Metadata Functions or something?

Reply:
Sure I will do that.

Reply:
I have updated it and built the documentation.